osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/openstack/securitygroup/MarkSecurityGroupMemberDeleteTask.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.core.broker.service.tasks.conformance.openstack.securitygroup;
+
+import javax.persistence.EntityManager;
+
+import org.apache.log4j.Logger;
+import org.osc.core.broker.model.entities.virtualization.SecurityGroupMember;
+import org.osc.core.broker.service.persistence.OSCEntityManager;
+import org.osc.core.broker.service.tasks.TransactionalTask;
+
+public class MarkSecurityGroupMemberDeleteTask extends TransactionalTask {
+
+    private static final Logger LOG = Logger.getLogger(MarkSecurityGroupMemberDeleteTask.class);
+
+    @Override
+    public String getName() {
+        return "Delete Security Group Member " + this.sgm.getMemberName() + "; id:" + this.sgm.getId();
```

Comment:
dont expose ID's if names are available.

Reply:
Names are good enough if unique. The unit tests relying on uniqueness of the task instance based on their names within a given graph.